docker.go

```diff
@@ -106,6 +106,11 @@ func (d *dockerContainerService) syncContainers(now time.Time) {
 			continue
 		}
 
+		if !container.State.Running {
+			log.Info("Skipping non-running container: id=", apiContainer.ID, " status=", container.State.Status)
```

Comment:
So, _technically_ we could still be hitting the race here, right? Especially if (for example) the starting container is the first in the list.

Reply:
Yeah, and the newest container *is* first in the list.  But:  I'm betting/wondering (without evidence) if the API call to `inspect` is going to either 1) provide non-cached live status or 2) slow enough to negate the race.  I've tried this multiple times in the test cluster and have yet to get the HTTP 500